metaci/testresults/models.py

```diff
@@ -1,24 +1,21 @@
 from __future__ import unicode_literals
 
-import os
-import logging
 import datetime
+import logging
+import os
 from collections import OrderedDict, namedtuple
-from dateutil.tz import gettz
 
+from dateutil.tz import gettz
+from django.apps import apps
 from django.db import models
-from django.db.models import F, Value, Avg, Count, Q, StdDev, FloatField
+from django.db.models import Avg, Count, F, FloatField, Q, StdDev, Value
 from django.db.models.functions import Cast
 from django.urls import reverse
 from django.utils import timezone
-from django.apps import apps
-
 from postgres_stats.aggregates import Percentile
 
-from metaci.testresults.choices import OUTCOME_CHOICES
-from metaci.testresults.choices import TEST_TYPE_CHOICES
-
 from metaci.build import models as build_models
+from metaci.testresults.choices import OUTCOME_CHOICES, TEST_TYPE_CHOICES
```

Comment:
Did you do this, or was it some automated import sorting in your editor? The rationale behind keeping them on separate lines is to avoid noise like this in diffs.

Reply:
I don't recall intentionally doing it.